<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>depgraphV::FilterPage</class>
 <widget class="QWidget" name="depgraphV::FilterPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>257</width>
    <height>200</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>257</width>
    <height>200</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QCheckBox" name="parseEnabled">
     <property name="text">
      <string notr="true"/>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QRadioButton" name="standardFiltersRadio">
        <property name="text">
         <string>Standard filters</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="standardFilters"/>
      </item>
      <item>
       <widget class="QRadioButton" name="customFilterRadio">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>21</height>
         </size>
        </property>
        <property name="text">
         <string>Custom (semicolon-separated)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="customFilters">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="inputMask">
         <string/>
        </property>
        <property name="text">
         <string notr="true"/>
        </property>
        <property name="placeholderText">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>customFilterRadio</sender>
   <signal>toggled(bool)</signal>
   <receiver>customFilters</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>145</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>184</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>customFilters</sender>
   <signal>editingFinished()</signal>
   <receiver>depgraphV::FilterPage</receiver>
   <slot>onCustomFiltersEditFinished()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>184</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>customFilters</sender>
   <signal>textChanged(QString)</signal>
   <receiver>depgraphV::FilterPage</receiver>
   <slot>onCustomFilterTextChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>184</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>parseEnabled</sender>
   <signal>toggled(bool)</signal>
   <receiver>frame</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>128</x>
     <y>19</y>
    </hint>
    <hint type="destinationlabel">
     <x>128</x>
     <y>99</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onCustomFiltersEditFinished()</slot>
  <slot>onCustomFilterTextChanged(QString)</slot>
 </slots>
</ui>
